.github/workflows/ci.yml

```diff
@@ -11,15 +11,21 @@ jobs:
           - platform: win32
             triplet: x86-windows
             arch: x86
-            vcpkgPackages: 'curl faad2 fluidsynth freetype fribidi libflac libjpeg-turbo libmad libmpeg2 libogg libpng libtheora libvorbis sdl2 sdl2-net zlib'
+            # fribidi is disabled due to timeouts when installing the package
+            configFlags: --enable-faad --enable-mpeg2 --disable-fribidi
+            vcpkgPackages: 'curl faad2 fluidsynth freetype libflac libjpeg-turbo libmad libmpeg2 libogg libpng libtheora libvorbis sdl2 sdl2-net zlib'
             useNasm: 'true'
           - platform: x64
             arch: x64
             triplet: x64-windows
-            vcpkgPackages: 'curl faad2 fluidsynth freetype fribidi libflac libjpeg-turbo libmad libmpeg2 libogg libpng libtheora libvorbis sdl2 sdl2-net zlib'
+            # fribidi is disabled due to timeouts when installing the package
+            configFlags: --enable-faad --enable-mpeg2 --disable-fribidi
+            vcpkgPackages: 'curl faad2 fluidsynth freetype libflac libjpeg-turbo libmad libmpeg2 libogg libpng libtheora libvorbis sdl2 sdl2-net zlib'
           - platform: arm64
             arch: arm64
             triplet: arm64-windows
+            # fribidi is disabled due to https://github.com/microsoft/vcpkg/issues/11248 [fribidi] Fribidi doesn't cross-compile on x86-64 to target arm/arm64
```

Comment:
Perhaps also note it's not sufficient to enable it here, as if I recall correctly, it's also excluded in create_project itself? Perhaps filing a ScummVM issue to track fribidi arm64 enablement on CI/vcpkg would be in order, especially with recent improvements to msvc cross-compilation with meson.

Reply:
OK, done.